ecs-cli/modules/utils/compose/convert_task_definition_test.go

```diff
@@ -1032,6 +1064,9 @@ func verifyUlimit(t *testing.T, output *ecs.Ulimit, name string, softLimit, hard
 }
 
 func convertToTaskDefinitionInTest(t *testing.T, name string, serviceConfig *config.ServiceConfig, taskRoleArn string, launchType string) *ecs.TaskDefinition {
+	volumeConfigs := make(map[string]*config.VolumeConfig)
+	volumeConfigs[namedVolume] = &config.VolumeConfig{}
```

Comment:
Why did you decide to add the volume config here? I'm thinking that it might be better to have it as an argument to `convertToTaskDefinitionInTest`, that way individual tests can use different values, and we can have some tests where the volume config is empty.

Reply:
Same reason why I didn't add `volumeConfigs` as argument to `convertToTaskDefinition`:
I didn't want to change the signature of existing methods, as I wasn't sure if this would be approved.
But I agree, makes sense to have it as an argument, I'll update it.